pkg/sgpp/utils.go

```diff
@@ -1,6 +1,14 @@
 package sgpp
```

Comment:
As a general pattern, I'd avoid `utils` packages as much as possible, and encapsulate the logic where its owned wherever possible. Of course, some things do need to be shared, but often there is a better named abstraction that encapsulates the concept. `utils` packages have a tendency to become a dumping ground for code, and become more complex and less maintainable over time.

Reply:
added algo/algo.go to utils folder. Seems ok practice referring to https://github.com/kubernetes/kubernetes/tree/master/pkg/util
Probably should use `util` instead of `utils` in VPC CNI?